<launch>

   <param name="/robot1_ip" type="str" value="192.168.1.126" />
   <param name="/robot2_ip" type="str" value="192.168.1.147" />
   <param name="/robot3_ip" type="str" value="192.168.1.135" />
   <param name="/robot4_ip" type="str" value="192.168.1.125" />
   <param name="/laptop_ip" type="str" value="192.168.1.127" />

   <node pkg="path_calculate" name = "udp_recv" type ="udp_server" output = "screen" />
   
   <node pkg="path_calculate" name = "udp_send" type ="udp_client" />
   








</launch>
